tests/integration/test_project_data_rows.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import uuid
+
+from labelbox import Project, Dataset
+
+
+def test_batches(project: Project, dataset: Dataset, image_url):
```

Comment:
Appreciate you adding a test, but do we need it? We already have tests/integration/test_batch.py

Reply:
Will remove